compiler/qsc/src/interpret/tests.rs

```diff
@@ -2230,5 +2230,43 @@ mod given_interpreter {
                 "#]],
             );
         }
+
+        #[test]
+        fn fuzz_2426() {
```

Comment:
There's got to be a better name. `recursive_type_constraint` maybe?

Reply:
I was following the naming convention we've used for the OpenQASM fuzz tests (see compiler/qsc_qasm/src/tests/fuzz.rs), but as those are in a fuzzing-specific file it makes more sense there. I personally like knowing right way when looking at the test that it was found via fuzzing and what the corresponding issue number is, but that can be accomplished with just a comment, so I'll adjust this test name to better fit the pattern in this file.